### PR TITLE
Added task files

### PR DESCRIPTION
**Pull request by kad-ep**
MR

### CODE REVIEW

**Comment by kad-ep** [comment]
please review

File: task3/task3/src/Main.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        String s1 =  getInput("Input first number: ");
+        String s2 =  getInput("Input second number: ");
+        int num1 = Integer.parseInt(s1);
+        int num2 = Integer.parseInt(s2);
+
+        System.out.println(num1 + " + " + num2 + " = " + addNumbers(num1,num2));
```
Comment (Stasnazarenko) [inline task3/task3/src/Main.java]: use String.format here and for such purposes please

File: task3/task3/src/Main.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        System.out.println(num1 + " * " + num2 + " = " + multiplyNumbers(num1,num2));
+        System.out.println(num1 + " / " + num2 + " = " + divideNumbers(num1,num2));
+        System.out.println(num1 + " mod " + num2 + " = " + divideNumbers(num1,num2));
+
+    }
+
+    private static String getInput(String prompt){
+        System.out.println(prompt);
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
```
Comment (Stasnazarenko) [inline task3/task3/src/Main.java]: 20 and 21st lines could be done in one line

File: task3/task3/src/Main.java (hunk 1 above)
Comment (Stasnazarenko) [reply task3/task3/src/Main.java]: done

**Review by Stasnazarenko — APPROVED** [review]

**Pull request merged**